src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/AuthorizationRepositoryTests.kt

```diff
@@ -126,4 +155,41 @@ class OrderlyWebAuthorizationRepositoryTests : CleanDatabaseTests()
                 .hasMessageContaining("Unknown user-group : 'nonsense'")
     }
 
+    @Test
+    fun `can add user to group`()
+    {
+        val sut = OrderlyAuthorizationRepository()
+        sut.createUserGroup("somegroup")
+        JooqContext().use {
+            it.dsl.insertInto(ORDERLYWEB_USER)
+                    .set(ORDERLYWEB_USER.EMAIL, "user@email.com")
+                    .set(ORDERLYWEB_USER.USER_SOURCE, "GitHub")
+                    .set(ORDERLYWEB_USER.USERNAME, "user.name")
+                    .execute()
+        }
+        sut.ensureGroupHasMember("somegroup", "user@email.com")
```

Comment:
Could also check the user-group-user row was actually added to the db.
Oh, although I see you are checking that in the CLI tests..

Reply:
Oh yeah, that should go here!